### ISSUE TITLE
`style` attribute in `toDOM` method isn't parsed correctly

### ISSUE THREAD

**Issue by konstantinmuenster** [nytimes/react-prosemirror#96]
When having a schema that defines a mark like this,

```ts
  underline: {
    parseDOM: [{ tag: "u" }],
    toDOM() {
      return ["span", { style: "text-decoration: underline" }, 0]
    },
  } as MarkSpec,
```

it fails to parse the style string correctly. It expects a JSX-like style object (`{ textDecoration: "underline" }`).

**Comment by plaisted** [comment]
I just ran into this as well when experimenting with the new branch. In some ways it's nice because it's more "reacty" but completely breaks compatibility with nodes / marks built on standard prosemirror.

**Comment by smoores-dev** [comment]
This is resolved by `0.7.0-next.8`!
